src/server/bg_services/buckets_reclaimer.js

```diff
@@ -44,6 +47,7 @@ class BucketsReclaimer {
                     await this.client.bucket.delete_bucket({ name: bucket.name, internal_call: true }, {
                         auth_token: auth_server.make_auth_token({
                             system_id: system_store.data.systems[0]._id,
+                            account_id: support_account._id,
```

Comment:
why do you use the support account? seems a bit unrelated here

maybe we can use the bucket owner?

```suggestion
                            account_id: bucket.owner_account._id,
```
just need to make sure that the account is not deleted before

Reply:
What if the account is deleted?
I've always got the support, so I'm not worried about it being deleted.

Reply:
using the support is good as a quick fix, not sure it is what the support account is meant to be.
try to see if the bucket owner can be used and is not deleted. if you're not sure then just leave the support account for now

Reply:
how about systeme.owner account?

Reply:
that was my suggestion as well, since we always have owner and it also makes sense